tests/test_routers.py

```diff
@@ -44,7 +55,7 @@ async def ignore_message(_: MessageType) -> None:
     return
 
 
-async def test_to_addr_router_setup(to_addr_router):
+async def test_to_addr_router_setup1(to_addr_router):
```

Comment:
Why the rename?

Reply:
I'll undo this. I use it to single out tests when running `py.test -k test_to_addr_router_setup1` to exclude all `test_to_addr_router_setup_*` from running